microsoft-edge/webview2/concepts/apis-capabilities.md

```diff
@@ -0,0 +1,1324 @@
+---
+title: Overview of WebView2 APIs and their capabilities
+description: Overview of WebView2 APIs and their capabilities.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/09/2022
+---
+# Overview of WebView2 APIs and their capabilities
+
+This article provides:
+*  A high-level understanding of the capabilities of the WebView2 technology.
+*  Concepts about what you can do with the WebView2 APIs.
+*  High-level description of WebView2 technology for app developers.
+*  Better fundamentals + understanding of WebView2 technology.
+
+Audience: App developers who are curious about the WebView2 technology and would like to know if WebView2 is for them.
+
+
+<!-- ====================================================================== -->
+## High-level interfaces spanning multiple areas
+
+This article categorizes the WebView2 interfaces, at a high level, for WebView2.  This article also lists selected members of interfaces.  These interfaces contain properties, methods, and events that span multiple categories in this article:
+
+#### [C#](#tab/c-sharp)
+
+* [CoreWebView2 Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2)
+* [CoreWebView2Controller Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2controller)
+* [CoreWebView2Environment Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment)
+* [CoreWebView2Settings Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2settings)
+
+#### [C++](#tab/cpp)
+
+* [ICoreWebView2](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2)
+* [ICoreWebView2Controller](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2controller)
+* [ICoreWebView2Environment](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment)
+* [ICoreWebView2Settings](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2settings)
+
+---
+
+Cross-platform API implementation: Most of the WebView2 APIs are initially developed for C++, and then most of the C++ APIs are wrapped as C# APIs.  This way, there is a consistent parallelism and equivalence across platforms and programming languages.
+<!-- link prefix: https://docs.microsoft.com -->
+
+
+### Overview of the top-level classes
```

Comment:
@aluhrs13 @david-risney - pls review this section

Reply:
Looks great!

Reply:
In the APIs Overview article, we'll shorten this section and continue to link to the new .md article that has the details from Dave.  

I plan to craft a "paragraph"/summary block that literally gets copied/sync'd back and forth between this section of the APIs Overview article and the new dedicated detailed article  https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/environment-controller-core?branch=pr-en-us-1971&tabs=c-sharp